lib/commands/performance.js

```diff
@@ -129,8 +129,8 @@ commands.mobileStartPerfRecord = async function mobileStartPerfRecord (opts = {}
   });
   proc.on('output', (stdout, stderr) => {
     (stdout || stderr).split('\n')
-      .filter(x => x.length)
-      .map(x => log.debug(`[instruments] ${x}`));
+      .filter((x) => x.length)
```

Comment:
Please don't change that, so we don't get a merge conflict

Reply:
Sure. I won't merge until your PR is in.